recipes-bsp/imx-atf/imx-atf_2.4.bb

```diff
@@ -27,6 +27,8 @@ EXTRA_OEMAKE += " \
 
 BUILD_OPTEE = "${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'true', 'false', d)}"
 
+CFLAGS_remove_mx8mq = "-O2"
```

Comment:
I've got a ticket open for a better fix if you want to wait for this.

Reply:
Actually, looks like this fix is fine, the problem is the 8M Quad has limited OCRAM. I updated the commit message.